src/main/java/pl/marcin/project/configuration/TomTomConfiguration.java

```diff
@@ -0,0 +1,29 @@
+package pl.marcin.project.configuration;
+
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.reactive.function.client.WebClient;
+
+@Configuration
+public class TomTomConfiguration {
+    private final String baseUrl = "https://api.tomtom.com";
```

Comment:
good you extracted it to a constant! we could take it one step further and read the data from `application.properties`. It will allow us to then freely test it by providing a fake "url" (not actually tomtom page) so that the tests would not call the real endpoint :)